### DISCUSSION TITLE
If scripting works in PowerShell 7

### DISCUSSION THREAD

**Discussion by edward-encoord** [dotnet/ResXResourceManager#500] [Q&A] [answered]
Thanks for this neat tool. I'm using it in Visual Studio and PowerShell.

It seems the scripting only works in PowerShell 5. When I run the same script in PowerShell 7, it can add the type from DLL, but says:

```
     |  $theHost = New-Object -TypeName ResXManager.Scripting.Host
     |             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Exception calling ".ctor" with "0" argument(s): "Could not load type
     | 'System.Runtime.Remoting.RemotingServices' from assembly 'mscorlib, Version=4.0.0.0, Culture=neutral,
     | PublicKeyToken=b77a5c561934e089'."
```

I'm wondering if someone can give me some hint on how to run this tool in PowerShell 7.

**Comment by tom-englert** [comment, marked as answer]
I think it's because the new PS is build on NetCore, so it can't load the NetFramework assemblies.
Can you try one of these, according to what I read it should be working with `netcoreapp3.1`

[Scripting.zip](https://github.com/dotnet/ResXResourceManager/files/9794559/Scripting.zip)

**Comment by edward-encoord** [comment]
I confirm `netcoreapp3.1` works on my machine. Thank you very much.
